modules/cli/src/integrationTest/java/org/apache/ignite/internal/NodeConfig.java

```diff
@@ -44,27 +44,25 @@ public class NodeConfig {
      * @return Config pattern.
      */
     public static String restSslBootstrapConfig(@Nullable String ciphers) {
-        return "{\n"
-                + "  network: {\n"
-                + "    port: {},\n"
-                + "    nodeFinder: {\n"
-                + "      netClusterNodes: [ {} ]\n"
-                + "    },\n"
-                + "  },\n"
-                + "  clientConnector.port: {} ,\n"
-                + "  rest: {\n"
+        return "ignite {\n"
+                + "  network {\n"
+                + "    port: {}\n"
+                + "    nodeFinder.netClusterNodes: [ {} ]\n"
+                + "  }\n"
+                + "  clientConnector.port: {}\n"
+                + "  rest {\n"
```

Comment:
Changing the rest of these lines wasn't really necessary, what motivated you to do more changes than you really needed?
It's everywhere in this PR, I bet it can be 3 times smaller if you remove all unnecessary changes